frontend/src/services/axiosConfig.js

```diff
@@ -0,0 +1,12 @@
+import axios from 'axios'
+
+const axiosInstance = axios.create()
+
+function setAxiosAuthToken(token) {
+  axiosInstance.defaults.headers.common.Authorization = 'Bearer ' + token
+}
+
+export default {
+  axiosInstance,
+  setAxiosAuthToken,
+}
```

Comment:
I prefer this way
```suggestion
export const axiosInstance = axios.create()

export function setAxiosAuthToken(token) {
  axiosInstance.defaults.headers.common.Authorization = 'Bearer ' + token
}
```

but thats really a matter of taste